src/components/AddItemForm.tsx

```diff
@@ -1,22 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+interface ListStateType {
+  taskList: string[];
+  setTaskList: (value: string[]) => void;
+}
+function AddItemForm({ taskList, setTaskList }: ListStateType) {
+  const [inputValue, setInputValue] = useState<string>('');
+
+  const handleInputText = (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
Acho que o nome dessa função mudou né? Ela não lida com evento de input text, mas sim o onSubmit.
```suggestion
  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
```